docs/ja/installation.md

```diff
@@ -0,0 +1,39 @@
+# インストール
+
+## 直接ダウンロード
+
+[dist フォルダ](https://github.com/vuejs/vue-validator/tree/dev/dist)を参照してください。dist フォルダにあるファイルは常に最新の安定版であることに注意してください。 - `dev` ブランチのソースは最新ではありません。
+
+## CDN
+
+### jsdelivr
+
+```html
+<script src="https://cdn.jsdelivr.net/vue.validator/2.1.1/vue-validator.min.js"></script>
+```
+
+## NPM
+
+### 安定版
+
+    $ npm install vue-validator
+
+### 開発版
+
+    $ git clone https://github.com/vuejs/vue-validator.git node_modules/vue-validator
+    $ cd node_modules/vue-validator
+    $ npm install
+    $ npm run build
+
+CommonJS で使用されるとき、明示的に `Vue.use()` 経由でバリデーターをインストールしなければなりません:
+
+> :warning: もし `vue-router` を使用している場合は、 `router#start` の前に `Vue.use()` でインストールしなければなりません。
```

Comment:
I was using `2.1.3`. (node v6.0.0 installed by nodebrew) At least I can work on translation even if I can't see the icon on localhost. Thanks for helping!